ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -602,3 +629,11 @@ func SortedGoString(v interface{}) (string, error) {
 	}
 	return string(b), nil
 }
+
+func areContainersAllEssential(ecsContainerDefs ContainerDefs) bool {
+	// NOTE: This is a silly hack for now, since the only field in a
```

Comment:
Can you either change this comment or add a new one that explains why this function is necessary. I found this comment a bit confusing.